packages/blade/scripts/publish-to-npm.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
```

Comment:
Suggestion: you could copy some of the background from PR description to this file for future context.